src/contributors/03-docker-development.md

```diff
@@ -23,6 +23,67 @@ Get the code with submodules:
 git clone https://github.com/LemmyNet/lemmy --recursive
 ```
 
+### Building
+
+Use these commands to create a custom container based on your local branch and tagged accordingly.
+
+This is useful if you want to modify the source code of your instance to add some extra functionalities which are not available in the main release.
+
+```bash
+sudo docker build . -f docker/Dockerfile -t "lemmy:${git rev-parse --abbrev-ref HEAD}"
```

Comment:
The `git rev-parse` looks a bit complicated, I would consider simply putting something like `lemmy:my-tag`.

Reply:
it's meant for a simple copy-paste. It doesn't matter if it's complicated as it does the job ;)